main.py

```diff
@@ -11,22 +11,18 @@ def main() -> None:
     for player_name, info in players.items():
         if not Race.objects.filter(
                 name=info["race"]["name"]).exists():
```

Comment:
If you use get_or_create no need .exists. It will improve code efficiency. Check in all places